front-end/src/Pages/Login/index.jsx

```diff
@@ -1,29 +1,116 @@
-import React from 'react';
 import './login.css';
-import '../../Assets/Styles/index.css';
-import logo from '../../Assets/Images/els.png';
+import _ from 'lodash';
```

Comment:
```suggestion
import size from 'lodash/size';
...
size(error)
```
Loadash is kinda big and you are importing it all. There is a little performance difference, but a little performance difference has a big impact on our app if the app will grow bigger and bigger.
The one that I suggested is only importing a loadash specific module which is a big factor in optimization.